### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by SanjoyanM**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -892,6 +893,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableCustomAppAuthentication
+
+Prevents apps using an ACS app-only access token to access SharePoint. Azure Access Control (ACS), a service of Azure Active Directory (Azure AD), has been retired on November 7, 2018. This retirement does not impact the SharePoint Add-in model, which uses the https://accounts.accesscontrol.windows.net hostname (which is not impacted by this retirement). For new tenants, apps using an ACS app-only access token is disabled by default. We recommend using the Azure AD app-only model which is modern and more secure.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:898]:
```suggestion
Prevents apps using an Azure Access Control (ACS) app-only access token to access SharePoint. ACS, a service of Azure Active Directory (Azure AD), has been retired on November 7, 2018. This retirement does not impact the SharePoint add-in model, which uses the https://accounts.accesscontrol.windows.net hostname (which is not impacted by this retirement). For new tenants, apps using an ACS app-only access token are disabled by default. We recommend using the Azure AD app-only model which is modern and more secure.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SanjoyanM, kindly commit the suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**